funnel/views/membership.py

```diff
@@ -65,6 +65,18 @@ def new_member(self):
 
         if request.method == 'POST':
             if membership_form.validate_on_submit():
+                if not membership_form.user.data.email:
+                    # users without verified email cannot be members
+                    return (
+                        {
+                            'status': 'error',
+                            'message': _("User does not have a verified email address"),
```

Comment:
This change will conflict with the proposal we're considering: user must have one verified contact, an email address or a phone number.

I suggest making this a method on the user object itself: `has_verified_contact_info`

Reply:
It may also be prudent to flag this in the select2 widget itself, instead of erroring after the user submits. People will complain if names are missing, so instead show name (a) low priority, at bottom, or (b) crossed out or otherwise marked as unavailable.

Reply:
The verified contact info requirement should also extend to the founding member of an organization, meaning you can't create one until you've verified a phone or email.